pkg/template/types.go

```diff
@@ -118,6 +120,16 @@ type HyperkubePodHostMount struct {
 	ReadOnly bool
 }
 
+type MultiMasters struct {
+	// If set to true, k8scloduconfig will render master template for cluster of 3 masters.
```

Comment:
Godoc should start with the varuable name.

```suggestion
	// Enabled when set to true will cause rendering master template for cluster of 3 masters. Single master otherwise.
```